src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -1173,7 +1173,7 @@ function isJsonCell(value: ICellValue): boolean {
 
 function isXmlCell(value: ICellValue): boolean {
 	let isXML = false;
-	if (value && !value.isNull) {
+	if (value && !value.isNull && value.displayValue.trim() !== '') {
 		parseXMLString(value.displayValue, (err, _) => {
```

Comment:
should the parseXMLString function treat spaces as valid xml?

Reply:
I don't think so, only valid xml contents should be allowed, that have open/close tags which `parseXMLString` can validate.

Reply:
Since `parseXMLString` is xml2js API, we can't modify it's logic, but we see it treated empty string as s valid XML, which depends if you accept empty values as valid types, but in our case it shouldn't have been the case..